pulsar-common/src/main/java/org/apache/pulsar/common/util/TrustManagerProxy.java

```diff
@@ -74,11 +76,16 @@ private void updateTrustManager() throws CertificateException, KeyStoreException
             FileNotFoundException, IOException {
         CertificateFactory factory = CertificateFactory.getInstance("X.509");
         try (InputStream inputStream = new FileInputStream(certFile.getFileName())) {
-            X509Certificate certificate = (X509Certificate) factory.generateCertificate(inputStream);
-            String alias = certificate.getSubjectX500Principal().getName();
             KeyStore keyStore = KeyStore.getInstance("JKS");
```

Comment:
Any particular reason JKS is hardcoded instead of using the JVM's default type?

```suggestion
            KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
```

Reply:
We should pass the KeyStore type to `TrustManagerProxy`, right now we are using JKS.